fitbenchmarking/core/fitting_benchmarking.py

```diff
@@ -36,13 +50,62 @@ def fitbenchmark_group(group_name, options, data_dir):
              list of failed problems and dictionary of unselected minimizers
     :rtype: tuple(list, list, dict)
     """
-    grabbed_output = output_grabber.OutputGrabber(options)
 
     # Extract problem definitions
     problem_group = misc.get_problem_files(data_dir)
 
-    results = []
+    #################################
+    # Loops over benchmark problems #
+    #################################
+    results, failed_problems, unselected_minimzers = \
+        loop_over_benchmark_problems(problem_group, options)
+
+    for keys, minimzers in unselected_minimzers.items():
+        minimizers_all = options.minimizers[keys]
+        options.minimizers[keys] = list(set(minimizers_all) - set(minimzers))
+
+    # options.minimizers = converge_minimizers
```

Comment:
Did you mean to leave `# options.minimizers = converge_minimizers` here?

Reply:
Good spot, I have removed this